### ISSUE TITLE
[Feature Request] Mouse/Touch Support on video seekbar

### ISSUE THREAD

**Issue by rezpower** [yuliskov/SmartTube#645]
Seeking will be much faster, easier and more precise with mouse. Also I am using smarttube on my phone. It's working perfectly fine but there is no way I can seek (go forward/backward) in the video. Touching the screen on seekbar causes video to pause.
Please look into this if possible.

**Comment by costinmx** [comment]
Indeed, it will be nice to have control over seekbar. Not possible now.

**Comment by rezpower** [comment]
> 
> 
> Indeed, it will be nice to have control over seekbar. Not possible now.

Thanks a lot for your reply. This app is already a great gift and heaven sent for android TV. Please keep this request somewhere on your mind to add it in future if a solution is available :)
